mmv1/products/cloudbuildv2/product.yaml

```diff
@@ -18,6 +18,9 @@ versions:
   - !ruby/object:Api::Product::Version
     name: beta
     base_url: https://cloudbuild.googleapis.com/v2/
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://cloudbuild.googleapis.com/v2/
```

Comment:
What's the difference between the beta & GA versions of the resource? Are there features that are still beta-only here?

Reply:
no differences at the moment. I left the beta for consistency with our cloudbuild (v1) product/resources. Should I remove beta?

Reply:
Done. Removed beta entries.